avocado/utils/disk.py

```
@@ -460,8 +473,8 @@
     :param device_name: Device  name example like sda
     :return: list of IO scheduler
     """
-    with open(__sched_path(device_name), "r", encoding="utf-8") as fl:
-        return fl.read().translate(str.maketrans("[]", " ")).split()
+    names = open(__sched_path(device_name), "r", encoding="utf-8").read()
```

Ruff (0.12.2)

Use a context manager for opening files

(SIM115)